src/guides/v2.3/security/google-recaptcha.md

```diff
@@ -12,6 +12,29 @@ Google reCAPTCHA provides a greater level of security for both the storefront an
 
 Google reCAPTCHA reduces potential user error when entering a Captcha code and encourages cart conversion without adding hurdles during checkout.
 
+## Types of Google reCAPTCHA
+
+There are three different types of reCAPTCHA available. They are,
+
+-  reCAPTCHA v3
+-  reCAPTCHA v2 ("I'm not a robot" Checkbox)
+-  reCAPTCHA v2 (Invisible reCAPTCHA badge)
+
+### reCAPTCHA v3
+
+reCAPTCHA v3 allows you to verify if an interaction is legitimate without any user interaction. It is a pure JavaScript API returning a score, giving you the ability to take action in the context of your site. For instance, requiring additional factors of authentication, sending a post to moderation, or throttling bots that may be scraping content.
+
+### reCAPTCHA v2 ("I'm not a robot" Checkbox)
+
+The "I'm not a robot" Checkbox requires the user to click a checkbox indicating the user is not a robot. This will either pass the user immediately (with No CAPTCHA) or challenge them to validate whether or not they are human. This is the simplest option to integrate with and only requires two lines of HTML to render the checkbox.
```

Comment:
```suggestion
The "I'm not a robot" checkbox requires the user to click a checkbox to prove that a human is interacting with the website. This will either pass the user immediately (with No CAPTCHA) or challenge them to validate whether or not they are human. This is the simplest option to integrate and only requires two lines of HTML to render the checkbox.
```